modules/hydrodyn/src/HydroDyn_Input.f90

```diff
@@ -364,9 +364,19 @@ SUBROUTINE HydroDyn_ParseInput( InputFileName, OutRootName, FileInfo_In, InputFi
       END IF
           
       DO I = 1,InputFileData%Morison%NAxCoefs
-            ! read the table entries   AxCoefID   CdAx  CaAx    in the HydroDyn input file
+         ! read the table entries AxCoefID, AxCd, AxCa, AxCp, AxFdMod, AxVnCOff, AxFDLoFSc in the HydroDyn input file
+         ! Try reading in 7 entries first
          call ParseAry( FileInfo_In, CurLine, ' axial coefficients line '//trim( Int2LStr(I)), tmpReArray, size(tmpReArray), ErrStat2, ErrMsg2, UnEc )
-            if (Failed())  return;
+         if ( ErrStat2 /= 0 ) then ! Try reading in 5 entries
+            tmpReArray(6) = -1.0  ! AxVnCoff
+            tmpReArray(7) =  1.0  ! AxFDLoFSc
+            call ParseAry( FileInfo_In, CurLine, ' axial coefficients line '//trim( Int2LStr(I)), tmpReArray(1:5), 5, ErrStat2, ErrMsg2, UnEc )
```

Comment:
Does `CurLine` not get incremented if there was an error during `ParseAry`, or do we need to add `CurLine=CurLine-1` before the next attempt?

Reply:
`ParseAry` returns before incrementing the line number in case of an error: https://github.com/OpenFAST/openfast/blob/5d6fd4f2bf45291cd6627bf6794dd466cf88edfa/modules/nwtc-library/src/NWTC_IO.f90#L3123-L3190